### PR TITLE
Add website, location, and github to organizations

### PR DESCRIPTION
**Pull request by mariobalca**
# Changes proposed ✍️
- Add the following columns to the `organizations` database table:
  - `website TEXT NULL`
  - `location TEXT NULL`
  - `github JSONB NULL`

**For the reviewer:** After taking a look at the `organizationService.ts` I noticed that we were using the **url** in a quite a few different places and use cases (mostly related to enrichment), so I've decided to simply add new columns to fill in the gaps we have with the mocks/frontend. Please let me know if you guys think we should not move forward like this.

## Checklist ✅
- [x] Label appropriately with `type:feature 🚀`, `type:enhancement ✨`, `type:bug 🐞`, or `type:documentation 📜`.
- [ ] Tests are passing.  
- [ ] New backend functionality has been unit-tested.
- [ ] Environment variables have been updated
  - [ ] Front-end: `frontend/.env.dist`
  - [ ] Backend: `backend/.env.dist`, `backend/.env.dist.staging`, `backend/.env.dist.staging`.
  - [ ] [Configuration docs](https://docs.crowd.dev/docs/configuration) have been updated.
  - [ ] Team members only: update environment variables in Password manager and update the team
- [ ] API documentation has been updated (if necessary) (see [docs on API documentation](https://docs.crowd.dev/docs/updating-api-documentation)).  
- [ ] [Quality standards](https://github.com/CrowdDotDev/crowd-github-test-public/blob/main/CONTRIBUTING.md#quality-standards) are met.  
- [ ] All changes have been tested in a staging site.  
- [ ] All changes are working locally running crowd.dev's Docker local environment.

### CODE REVIEW

File: backend/src/serverless/integrations/webhooks/github.ts [hunk 1]
```diff
@@ -379,6 +379,7 @@ export default class GitHubWebhook {
               location: fromAPI.location ?? null,
               logo: fromAPI.avatarUrl ?? null,
               url: fromAPI.url ?? null,
+              github: { handle: fromAPI.resourcePath },
```
Comment (epipav) [inline backend/src/serverless/integrations/webhooks/github.ts:382]: shouldn't this be `fromApi.url` ? The graphql response has the gh organization link in `url`

Comment (epipav) [inline backend/src/serverless/integrations/webhooks/github.ts:382]: shouldn't this be `fromApi.url` ? The graphql response has the gh organization link in `url`

**Review by epipav — CHANGES_REQUESTED** [review]
Looking good! 
We also need to add the change in organanization parse to `githubIntegrationService`

File: backend/src/serverless/integrations/webhooks/github.ts (hunk 1 above)
Comment (joanagmaia) [reply backend/src/serverless/integrations/webhooks/github.ts]: At least from sample data the url is different from the handles. Handles should indicate platform profiles/handlers right? For instance, for crowd, the url could be `https://www.crowd.dev/` and the github handle should be `CrowdDotDev`

**Review by epipav — APPROVED** [review]

**Pull request merged**